packages/psss-server/src/app/addRoutesToApp.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Express, Request, Response, NextFunction } from 'express';
+import { InternalServerError, RespondingError } from '@tupaia/utils';
+import { PsssRequest } from '../types';
+import { LoginHandler, LogoutHandler, TestHandler } from '../routes';
+import { RouteHandler } from '../routes/RouteHandler';
+
+const handleWith = (Handler: typeof RouteHandler) => (
```

Comment:
Oh this is really cool, I didn't realise you could use `typeof` on a class like this 👍